Tella/Supporting Files/ar.lproj/Localizable.strings

```diff
@@ -78,6 +78,9 @@
 "Vault_RenameFile_SheetTitle" = "إعادة تسمية الملف";
 "Vault_RenameFile_Cancel_SheetAction" = "CANCEL";
 "Vault_RenameFile_Save_SheetAction" = "SAVE";
+"Vault_Confirm_delete_Description" = "Some of the selected files are currently used in your connections. Deleting may impact forms and reports that haven’t been sent yet";
```

Comment:
What do you think if you change these lacalizable to :
`"Vault_DeleteFile_Warning_SheetExpl"` 
`"Vault_DeleteFile_Warning_SheetTitle" `
`"Vault_DeleteFile_DeleteAnyway_SheetAction"  `

Reply:
and maybe this  "Vault_DeleteFile_Warning_SheetDescription" instead this "Vault_DeleteFile_Warning_SheetExpl"